solid/src/main/java/com/inrupt/client/solid/SolidContainer.java

```diff
@@ -145,22 +141,43 @@ static Predicate<Triple> verifyContainmentTriple(final String container) {
     }
 
     static boolean verifyContainmentIri(final String container, final IRI object) {
-        if (!object.getIRIString().startsWith(container)) {
-            // Out-of-domain containment triple object
+
+        // 1 URI Structure Tests
```

Comment:
What do these numbers refer to? I looked in several place I imagined would be relevant, but haven't found anything like this.

Reply:
That was more for my own organization. I've backed that out